site/pages/admin/user-management.page.tsx

```diff
@@ -78,42 +68,37 @@ const UserManagement = ({ userList, csrfToken }: UserManagementPageProps): React
         userGroup: string,
         showResendInvite?: boolean,
     ) => {
-        {
-            if (userGroup !== "system-admins") {
-                return (
+        return (
+            <>
+                {showResendInvite ? (
```

Comment:
Could System admin users benefit from having resent invite incase of unregistered status. If the invitation for a system admin user has expired then there should be an option to resend the invite.

In case of system admins who are no longer part of the team - there should be an option to delete them

Reply:
Spoke with Chris yesterday about this, he confirmed that system admins and org admins should only be able to view system admins and perform no actions. 

I think there will be a separate ticket about system admin functionality as ideally we don't want them to be a part of an organisation at all.